include/sys/Time.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Time.h - Windows compatible implementation of sys/time.h
+ * Copied from http://www.codefull.net/2015/12/systime-h-replacement-for-windows/
+ *
+ * Copyright (c) 2004-2008 Tobias Doerffel <tobydox/at/users.sourceforge.net>
```

Comment:
This copyright notice is wrong. This is important to get right - we don't want any legal trouble. Try to track down the original copyright declaration and include it here.